ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-ospf.j2

```diff
@@ -120,6 +120,44 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{# OSPF Timers #}
+{%     set has.found = false %}
+{%     for process in router_ospf.process_ids %}
```

Comment:
missed this last time. Please add `| arista.avd.natural_sort` to this `for` so we catch if not defined.